src/c++/perf_analyzer/README.md

```diff
@@ -73,6 +73,97 @@ Inferences/Second vs. Client p95 Batch Latency
 Concurrency: 1, throughput: 69.6 infer/sec, latency 19673 usec
 ```
 
+## Quick Start
+
+The steps below will guide you through using Perf Analyzer to profile a simple
+Tensorflow model: `simple`.
+
+### Step 1: Start Triton Container
+
+```bash
+export RELEASE=<yy.mm> # e.g. for February of 2023, do `export RELEASE=23.02`
```

Comment:
I have a concern that if a user went right now, they would say "oh it is march 2023 right now" and put in 23.03, which has not been released.

Maybe this?
# e.g. to use the release from the end of February 2023, do `export RELEASE=23.02`

Reply:
no idea why that comment is yelling ....

Reply:
Reworded 👍